pkg/adapter/nimbus-kubearmor/processor/kspbuilder.go

```diff
@@ -184,6 +205,166 @@ func swDeploymentToolsKsp() kubearmorv1.KubeArmorPolicy {
 	}
 }
 
+func disallowCapabilities() kubearmorv1.KubeArmorPolicy {
+	return kubearmorv1.KubeArmorPolicy{
+		Spec: kubearmorv1.KubeArmorPolicySpec{
+			Capabilities: kubearmorv1.CapabilitiesType{
+				MatchCapabilities: []kubearmorv1.MatchCapabilitiesType{
+					{
+						Capability: "sys_admin",
+					},
+					{
+						Capability: "sys_ptrace",
+					},
+					{
+						Capability: "sys_module",
+					},
+					{
+						Capability: "dac_read_search",
+					},
+					{
+						Capability: "dac_override",
+					},
+				},
+			},
+		},
+	}
+}
+
+func disallowChRoot() kubearmorv1.KubeArmorPolicy {
+	return kubearmorv1.KubeArmorPolicy{
+		Spec: kubearmorv1.KubeArmorPolicySpec{
+			Process: kubearmorv1.ProcessType{
+				MatchPaths: []kubearmorv1.ProcessPathType{
+					{
+						Path: "/usr/sbin/chroot",
+					},
+					{
+						Path: "/sbin/chroot",
+					},
+				},
+			},
+		},
+	}
+}
+
+
+func disallowMatchPath() kubearmorv1.KubeArmorPolicy {
```

Comment:
Isn't it possible to leverage [swDeploymentTools](https://github.com/5GSEC/nimbus/blob/b91563b0ccd36deee1c6afd73cf51fd2aa25aae8/pkg/adapter/nimbus-kubearmor/processor/kspbuilder.go#L83) policy by adding `wget` and `curl` executables path there?

E.g.,
```go
...
case idpool.DisallowChRoot:
    policy :=  swDeploymentToolsKsp()
    policy.Spec.Process.MatchPaths = append(ksp.Spec.Process.MatchPaths, wget-path, curl-path)
    return policy
...
```